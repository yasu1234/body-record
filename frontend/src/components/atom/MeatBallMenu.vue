<template>
  <div class="meatball-menu">
    <div v-for="i in 3" class="dot" />
  </div>
</template>

<script setup></script>

<style scoped>
.meatball-menu {
  width: 20px;
  height: 10px;
  display: flex;
  flex-direction: row;
  justify-content: space-between;
  align-items: center;
  cursor: pointer;
}
.dot {
  width: 3px;
  height: 3px;
  background-color: #333;
  border-radius: 50%;
}
</style>
