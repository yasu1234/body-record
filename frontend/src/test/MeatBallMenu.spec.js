import { shallowMount } from "@vue/test-utils";
import MeatBallMenu from "@/components/atom/MeatBallMenu.vue";

describe("MeatBallMenu", () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallowMount(MeatBallMenu);
  });

  afterEach(() => {
    jest.clearAllMocks();
  });

  it("SnapShotテスト初期表示", () => {
    expect(wrapper.element).toMatchSnapshot();
  });
});
