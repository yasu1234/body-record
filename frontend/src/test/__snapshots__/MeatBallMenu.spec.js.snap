// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MeatBallMenu SnapShotテスト初期表示 1`] = `
<div
  class="meatball-menu"
>
  
  <div
    class="dot"
  />
  <div
    class="dot"
  />
  <div
    class="dot"
  />
  
</div>
`;
